packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -45,7 +46,7 @@ h| Description
 * Ensure data localization requirements are implemented in line with local regulations and contractual obligations
 
 | 1.7 Incident handling
-| * Notify relevant parties about any security breach that affects personal data without undue delay, no later than 72 hours upon discovery
+| * Notify relevant parties about any security breach that affects sensitive information no later than 72 hours upon discovery, and upon learning any additional details of the breach. Consider reporting the breach to relevant national cybersecurity agencies
```

Comment:
We may benefit from the reporting part being separate bullet point here. I adjusted the wording to make it read better stand-alone, but would appreciate other thoughts on wording here.

```suggestion
| * Notify relevant parties about any security breach that affects sensitive information no later than 72 hours upon discovery, and upon learning any additional details of the breach
* Consider reporting the breach to relevant national cybersecurity agencies in line with local guidance and regulations
```

Reply:
Looks good to me!